ykllvmwrap/src/ykllvmwrap.cc

```diff
@@ -235,7 +236,15 @@ extern "C" void *compileModule(string TraceName, Module *M,
     errx(EXIT_FAILURE, "Couldn't compile trace: %s",
          EE->getErrorMessage().c_str());
 
-  return (void *)EE->getFunctionAddress(TraceName);
+  // Allocate space for compiled trace address, stackmap address, and stackmap
+  // size (plus padding).
+  // FIXME This is a temporary hack until the redesigned hot location is up.
+  uintptr_t *ptr = (uintptr_t *) malloc(32);
```

Comment:
Let's at least make this `malloc(sizeof(uintptr_t) * 3)`.

Reply:
I assume that the other 8 bytes are the padding the comment mentions?

Reply:
What about the padding?

Reply:
Oh, seems like padding isn't needed for malloc.

Reply:
Fixed in f5ca58d.